client/src/components/graph/graph.tsx

```diff
@@ -368,24 +389,40 @@ class Graph extends React.Component<GraphProps, GraphState> {
 
   handleResize = (): void => {
     const viewport = this.getViewportDimensions();
-    const projectionTF = createProjectionTF(viewport.width, viewport.height);
+    const projectionTF = createProjectionTF({
+      viewportWidth: viewport.width,
+      viewportHeight: viewport.height,
+      isSpatialMode: isSpatialMode(this.props),
+    });
+
     this.setState((state) => ({
       ...state,
       viewport,
       projectionTF,
     }));
+
+    this.handleGoHome();
```

Comment:
on resize we'll re-center for now! The resize ticket is lowered to P1 per discussion with Nik and Seve:

https://app.zenhub.com/workspaces/single-cell-5e2a191dad828d52cc78b028/issues/gh/chanzuckerberg/single-cell-explorer/892